### ISSUE TITLE
Hyperparameter in IRN

### ISSUE THREAD

**Issue by ZhangYushan3** [allenwu97/Adaptive-Spatial-BCE-Loss-for-Weakly-Supervised-Semantic-Segmentation#1]
Hi! What threshold did you set as conf_fg_thres and conf_bg_thres in irn?

**Comment by allenwu97** [comment]
The default value in my code is 0.6 and 0.25. I'm not sure whether this is the value corresponding to the best result， because time is too long.

**Comment by ZhangYushan3** [comment]
Thanks for the reply. I got almost the same result for cams as the paper claimed (miou 70.4 with CRF). But when I train it on IRN official repo, I got miou 61.2 for the segmentation mask. Is there any other hyperparameter I should set before the IRN? For example 'sem_seg_bg_thres'?

**Comment by allenwu97** [comment]
My 'sem_seg_bg_thres' default value is 0.4.
And  i think `cams = torch.pow(cams, 1.5)` is a trick you should add in make_sem_seg_labels.py. I find the trick from other open source projects, and I remember it was very effective

**Comment by ZhangYushan3** [comment]
Thanks. This is very helpful!

**Comment by vtars** [comment]
Can you give more details about the code `cams = torch.pow(cams, 1.5)` ?  Such as a reference link? Thanks!

**Comment by allenwu97** [comment]
> Can you give more details about the code `cams = torch.pow(cams, 1.5)` ? Such as a reference link? Thanks!

https://github.com/jbeomlee93/AdvCAM/blob/main/step/make_sem_seg_labels.py#L42
